scripts/dev-release.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -eo pipefail
+#!/bin/bash -exo pipefail
```

Comment:
Probably best to drop the `x`. You should be ok though because IIRC when circle runs this script it'll just use `#!/bin/bash -eo pipefail`